Server/VikingVault.Services/BankAccountService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.DataAccess.Models;
+using VikingVault.Services.Abstractions;
+using VikingVault.Services.Exceptions;
+
+namespace VikingVault.Services
+{
+    public class BankAccountService : IBankAccountService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+
+        public BankAccountService(VikingVaultDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public BankAccount CreateBankAccount(BankAccount account)
+        {
+            try
+            {
+                _dbContext.Add(account);
+                _dbContext.SaveChanges();
+            }
+            catch (Exception e)
+            {
+                if (e is DbUpdateException || e is DbUpdateConcurrencyException) throw new BankAccountServiceException();
```

Comment:
Please try to avoid using one-liners statements, even though it's more lines of code to say

if () 
{
}
, it's easier to read and is more consistent and less prone to errors